sql/moz-fx-data-shared-prod/firefox_ios_derived/attributable_clients_v1/query.sql

```diff
@@ -0,0 +1,144 @@
+CREATE TEMP FUNCTION sum_map_values(map ARRAY<STRUCT<key STRING, value INT64>>)
+RETURNS INT64 AS (
+  (SELECT SUM(value) FROM UNNEST(map))
+);
+
+WITH client_day AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    sample_id,
+    client_info.client_id,
+    LOGICAL_OR(
+      mozfun.norm.extract_version(client_info.app_display_version, 'major') >= 107
+    ) AS has_search_data,
+    SUM(sum_map_values(metrics.labeled_counter.search_in_content)) AS searches,
+    SUM(sum_map_values(metrics.labeled_counter.browser_search_with_ads)) AS searches_with_ads,
+    SUM(sum_map_values(metrics.labeled_counter.browser_search_ad_clicks)) AS ad_clicks,
+  FROM
+    firefox_ios.baseline
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+  GROUP BY
+    submission_date,
+    sample_id,
+    client_id
+),
+searches AS (
+  SELECT
+    submission_date,
+    client_id,
+    LOGICAL_OR(mozfun.norm.extract_version(app_version, 'major') < 107) AS has_search_data,
+    SUM(search_count) AS searches,
+    SUM(search_with_ads) AS searches_with_ads,
+    SUM(ad_click) AS ad_clicks
+  FROM
+    search_derived.mobile_search_clients_daily_v1
+  WHERE
+    submission_date = @submission_date
+    -- #TODO: is there a specific app we should be filtering for here for search info for ios?
+    -- AND normalized_app_name = 'Klar'
+    AND os = 'iOS'
+  GROUP BY
+    submission_date,
+    client_id
+),
+first_seen AS (
+  SELECT
+    client_id,
+    country,
+    first_seen_date
+  FROM
+    firefox_ios.baseline_clients_first_seen
+  WHERE
+    submission_date >= "2021-01-01"  -- # TODO: should this be @submission_date AKA @submission_date?
+),
+adjust_client AS (
+  SELECT
+    client_id,
+    adjust_info.adjust_network,
+    adjust_info.adjust_ad_group AS adjust_adgroup,
+    adjust_info.adjust_campaign,
+    adjust_info.adjust_creative,
+    metadata.reported_first_session_ping,
+  FROM
+    firefox_ios.firefox_ios_clients
+  WHERE
+    adjust_info.adjust_network <> "Unknown"
+),
+activations AS (
+  SELECT
+    client_id,
```

Comment:
I found a few cases with more than one activation for Fx Android so the query [retrievs the latest one](https://github.com/mozilla/bigquery-etl/blob/2d18693cca528ac22ac456f5666964dba0186b54/sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/query.sql#L23-L32), you may want to consider that case as well?

Reply:
activations lives now inside `firefox_ios_clients_v1`. Marking this as resolved since this logic is no longer present.

As an additional note, I'm making a b2 of the activations table for ios and added a comment there to make sure that table only ever contains one entry per client.